src/pudl/etl/__init__.py

```diff
@@ -181,8 +188,14 @@ def _get_keys_from_assets(asset_def: AssetsDefinition | AssetSpec) -> list[Asset
     "ferc_to_sqlite_settings": ferc_to_sqlite_settings,
     "parquet_io_manager": parquet_io_manager,
     "geoparquet_io_manager": geoparquet_io_manager,
+    "ferceqr_extract_settings": pudl.extract.ferceqr.ExtractSettings(
+        archive=os.getenv(  # Default to read directly from GCS if local path not specified
+            "EQR_ARCHIVE_PATH", "gs://archives.catalyst.coop/ferceqr/published"
```

Comment:
For externally visible mentions, if we can stick to using the data source ID (`ferceqr`) rather than other abbreviations I think it will be easier to find related elements when searching later on.

```suggestion
            "FERCEQR_ARCHIVE_PATH", "gs://archives.catalyst.coop/ferceqr/published"
```